table/column_test.go

```diff
@@ -338,7 +338,7 @@ func (t *testTableSuite) TestGetDefaultValue(c *C) {
 				},
 			},
 			true,
-			types.Datum{},
+			types.NewIntDatum(0),
```

Comment:
Is this a bug fix?

Reply:
No, it a unit test for function `GetColDefaultValue`. It is changed due to the behavior of `GetColDefaultValue` was changed.